.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: required
```

Comment:
The reason I integrated with Travis was because I didn't have any luck with Quay builds. Also, I'm not sure what version of Docker Quay is using (this needs >=17).